examples/seismic/tti/operators.py

```diff
@@ -73,7 +73,6 @@ def Gzz_centered(model, field, costheta, sintheta, cosphi, sinphi, space_order):
     Rotated second order derivative w.r.t. z.
     """
     order1 = space_order // 2
-    x, y, z = model.space_dimensions
```

Comment:
no complaints from flake8? 😮

Reply:
Yeah I know right? WTH

Reply:
I mean should have complained before this is not used anywher

Reply:
Yeah it is why it's weird.